examples/simple_experiment.rs

```diff
@@ -27,7 +27,9 @@ fn main() {
     let pypy_bench = Benchmark::new("./benchmarks/binarytrees/binarytrees.py", &pypy)
         .tag("abstract_benchmark_name", "binarytrees");
     let lua_bench = Benchmark::new("./benchmarks/binarytrees/binarytrees.lua", &luajit)
-        .tag("abstract_benchmark_name", "binarytrees");
+        .tag("abstract_benchmark_name", "binarytrees")
+        .stack_lim(Limit::KiB(8.192))
```

Comment:
One thing we can think about for the future: perhaps the experiment-level limits are the defaults, which can then be overridden on a per-benchmark basis. Don't worry about that for this PR though.

Reply:
(and maybe for pexecs and iterations too? I dunno. Let's come back to that)

Reply:
Yeah, I guess we want that kind of flexibility. I can address this in a future PR.